applications/matter_bridge/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ target_include_directories(app PRIVATE
     src/bridged_device_types
     ${COMMON_ROOT}/src
     ${COMMON_ROOT}/src/bridge/
+    ${COMMON_ROOT}/src/bridge/util
```

Comment:
I think that such directory doesn't exist.

Reply:
Right, it sneaked in during rebasing. Now it's added.